README.md

```diff
@@ -15,17 +15,19 @@ As of our most recent design, the major components and technologies are as follo
 ## Tech Stack
 - [TypeScript](https://www.typescriptlang.org)
 - [Chakra UI](https://chakra-ui.com) 
-- [Prisma](https://www.prisma.io/)(?)
+- [Prisma](https://www.prisma.io/) (?)
 - [PostgresSQL](https://www.postgresql.org/)
 - [Sync Inc](https://syncinc.so/)
 - [Airtable](https://airtable.com)
 
 ## Requirements
 - [Node.js](https://nodejs.org/en/)
+- [Yarn 1](https://classic.yarnpkg.com/lang/en/)
 
 ## Running Locally
 - Copy this repo: `git clone https://github.com/MutualAidNYC/services-lists.git`
-- Install dependencies: `npm install`
+- Install the Yarn package manager (version 1): `npm install -global yarn`
+- Install dependencies: `yarn install`
```

Comment:
Switching to Yarn for package management because in my experience it's better at resolving weird dependency issues that come up when you upgrade/add packages.

Yarn also has versions 2 and up now so I'll look into migrating to a more recent version in another PR.